src/maps/process.js

```diff
@@ -110,7 +112,22 @@ function msa(level = 'msa') {
   return geo;
 }
 
-function counties(level = 'county') {
+async function generatePopulationLookup() {
+  const csv = csvParse(
+    await fetch(
+      'https://raw.githubusercontent.com/cmu-delphi/covidcast-indicators/main/_delphi_utils_python/delphi_utils/data/fips_pop.csv',
```

Comment:
Are there any cross-site considerations to be made when pulling this data from github directly?  Would it be better to pull the data down and package it as part of the site at build time?

Reply:
the raw data is just processed on demand. It will generate among others the CSV file that is also checked in.

Reply:
Ah so, this is only at build time?  That makes sense.  It'd be nice if there was a npm package that we could pull down as a first-class dependency, but until then this make sense.